include/gc/Transforms/Passes.td

```diff
@@ -32,6 +32,17 @@ def ConvertOneDNNGraphToLinalg : Pass<"convert-onednn-graph-to-linalg"> {
   ];
 }
 
+def ConvertMemRefToCPURuntime : Pass<"convert-memref-to-cpuruntime"> {
```

Comment:
Can't this be a `func::FuncOp` pass? It should just replace ops with appropriate calls.

Reply:
made it a `func::FuncOp` pass.